src/filesys.cpp

```diff
@@ -206,6 +208,8 @@ std::string TempPath()
 #include <sys/wait.h>
 #include <unistd.h>
 
+static const std::string bad_names[] = { "" };
```

Comment:
This is intentional, the first list is for windows and the second one is for linux/mac os.